src/com/maddyhome/idea/vim/group/MotionGroup.java

```diff
@@ -1315,6 +1347,14 @@ public int moveCaretGotoLineLast(@NotNull Editor editor, int rawCount, int line)
       .normalizeLine(editor, EditorHelper.getLineCount(editor) - 1) : line);
   }
 
+  public int moveCaretGotoLineLast(@NotNull Editor editor, int rawCount) {
```

Comment:
Is it a bug fix? If so, let's add a test and move it into a separate PR. Otherwise let's just discard this change.